synapse/lib/cell.py

```diff
@@ -2814,7 +2814,7 @@ def getCellNexsRoot(self):
 
     async def _initCellHiveAuth(self):
 
-        maxusers = self.conf.get('max:users')
+        maxusers = self.conf.get('max:users', 0)
```

Comment:
Is there a time when we would now have a `max:users` value when we have a `default` being set in confbase?

Reply:
Yep, this test: https://github.com/vertexproject/synapse/blob/blackout/SYN-5870/user-count-checks-archived-locked/synapse/tests/test_lib_config.py#L248

Reply:
That makes me think there is a separate error in the config stuff if the default isn't being set. Checking.